docs/trigger.md

```diff
@@ -0,0 +1,34 @@
+# How to trigger your task on new diffs
+
+## Requirements
+
+1. Your repository must use [Mozilla Phabricator's instance](https://phabricator.services.mozilla.com/)
+2. Your repository must use [Taskcluster as CI](https://docs.taskcluster.net) (at least one task must start on each push).
+3. You are adding a Taskcluster task that runs on each push, analyzes the modified source code and lists potential issues.
+
+## Step for mozilla-central
```

Comment:
I'd move this and NSS into `how to create a new analyzer`, and just have the `requirements` (which apply to when you want to hook up a new repo) to a separate file.
Basically having this structure:
```
SUMMARY
  - HOW TO ADD A NEW ANALYZER
    - TASK DEFINITION
    - ...
    - ANALYZER OUTPUT FORMAT
  - HOW TO HOOK UP A NEW REPOSITORY
```